packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -327,6 +328,7 @@ function Layer({ element }) {
               title={__('Delete Layer', 'web-stories')}
               hasTail
               isDelayed
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
```

Comment:
Since we are no longer changing these to BOTTOM_END we can leave it. BOTTOM is the default.

Reply:
so this line can be removed? Sorry if this is a note to yourself and I'm being redundant!

Reply:
Yep. I can remove it here and below.